### ISSUE TITLE
ComedyCentral module broken (extractor/mtv.py description_node.text is None)

### ISSUE THREAD

**Issue by smkent** [rg3/youtube-dl#8968]
This appears to have become an issue sometime within the past 2 days.

```
Traceback (most recent call last):
  File "/usr/bin/youtube-dl", line 6, in <module>
    youtube_dl.main()
  File "/usr/lib64/python2.7/site-packages/youtube_dl/__init__.py", line 412, in main
    _real_main(argv)
  File "/usr/lib64/python2.7/site-packages/youtube_dl/__init__.py", line 402, in _real_main
    retcode = ydl.download(all_urls)
  File "/usr/lib64/python2.7/site-packages/youtube_dl/YoutubeDL.py", line 1719, in download
    url, force_generic_extractor=self.params.get('force_generic_extractor', False))
  File "/usr/lib64/python2.7/site-packages/youtube_dl/YoutubeDL.py", line 668, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib64/python2.7/site-packages/youtube_dl/extractor/common.py", line 320, in extract
    return self._real_extract(url)
  File "/usr/lib64/python2.7/site-packages/youtube_dl/extractor/mtv.py", line 219, in _real_extract
    videos_info = self._get_videos_info(mgid)
  File "/usr/lib64/python2.7/site-packages/youtube_dl/extractor/mtv.py", line 183, in _get_videos_info
    return self._get_videos_info_from_url(info_url, video_id)
  File "/usr/lib64/python2.7/site-packages/youtube_dl/extractor/mtv.py", line 190, in _get_videos_info_from_url
    [self._get_video_info(item) for item in idoc.findall('.//item')])
  File "/usr/lib64/python2.7/site-packages/youtube_dl/extractor/mtv.py", line 135, in _get_video_info
    description = description_node.text.strip()
AttributeError: 'NoneType' object has no attribute 'strip'
```

Temporary workaround:

```
--- /usr/lib64/python2.7/site-packages/youtube_dl/extractor/mtv.py      2016-03-24 17:35:48.518820768 -0700
+++ /usr/lib64/python2.7/site-packages/youtube_dl/extractor/mtv.py      2016-03-24 17:32:24.162000946 -0700
@@ -132,7 +132,10 @@
 
         description_node = itemdoc.find('description')
         if description_node is not None:
-            description = description_node.text.strip()
+            if description_node.text:
+                description = description_node.text.strip()
+            else:
+                description = None
         else:
             description = None


```

**Comment by dstftw** [comment]
Already fixed.

**Comment by phihag** [comment]
I believe this issue is already fixed in youtube-dl 2016.03.25. If it still fails for you, please open a new issue and make sure to include the **complete** output of youtube-dl when called with `-v`.
